css-fonts-5/Overview.bs

```diff
@@ -149,21 +149,21 @@ the 'ascent-override', 'descent-override', 'line-gap-override', and 'size-adjust
 
 <pre class='descdef'>
 Name: ascent-override
-Value: normal | <<percentage>>
+Value: normal | <<percentage>>{1,2}
```

Comment:
Does the `{1,2}` here apply to the entire alternation `normal | <<percentage>>`, or only to the `<<percentage>>` (which is what it looks like to my naïve eye, at least)?

If the latter, I think that's not what we want, as it wouldn't allow things like `ascent-override: normal 75%`.